opt_einsum/tests/test_backends.py

```diff
@@ -36,6 +38,12 @@
 except ImportError:
     found_sparse = False
 
+try:
+    import torch
+    found_torch = True
+except ImportError:
+    found_torch = False
+
```

Comment:
Could try using `pytest.importorskip` or similar through here. Might make things a bit simpler.

Reply:
Yep that simplified things!